vendor/bat-native-ads/src/bat/ads/internal/p2a/p2a_ad_opportunities/p2a_ad_opportunity.cc

```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/p2a/p2a_ad_opportunities/p2a_ad_opportunity.h"
+
+#include <string>
+#include <vector>
+
+#include "base/strings/stringprintf.h"
+#include "bat/ads/ad_type.h"
+#include "bat/ads/internal/p2a/p2a.h"
+#include "bat/ads/internal/p2a/p2a_ad_opportunities/p2a_ad_opportunity_questions.h"
+
+namespace ads {
+namespace p2a {
+
+void RecordAdOpportunityForSegments(const AdType& ad_type,
+                                    const SegmentList& segments) {
+  const std::string type_as_string = std::string(ad_type);
+
+  const std::string name =
```

Comment:
this will break p2a and would require changing https://github.com/brave/brave-core/blob/671e91d50ac007cca9dfea3e30e755acb7b7d4e5/components/p3a/brave_p3a_service.cc#L103. I suggest we change for ad notifications and have other creative types not sent for now

Reply:
Discussed with @moritzhaller and all good